CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ With Python 3.8 or up, Node 16, and [pre-commit](https://pre-commit.com/),
 ```bash
 git clone git@github.com:torchbox/tbxforms.git
 cd tbxforms/
-pip install poetry==1.1.13
+pip install poetry==1.4.1
```

Comment:
Poetry has some [pretty specific instructions](https://python-poetry.org/docs/#installing-with-the-official-installer) for how it should be installed. Maybe we should remove this line, and a note about the poetry requirement above?